README.md

```diff
@@ -7,15 +7,76 @@
 [![Apache 2.0 Licensed][license-image]][license-link]
 ![Rust 1.39+][rustc-image]
 
-Rust crate for interacting with [Tendermint]: a high-performance blockchain
+Rust crates for interacting with [Tendermint]: a high-performance blockchain
 consensus engine that powers Byzantine fault tolerant applications written
 in any programming language.
 
-[Documentation][docs-link]
+Also includes [TLA+ specifications](/docs/spec).
 
-## Requirements
+## Releases
 
-- Rust 1.39+
+Release tags can be found on
+[Github](https://github.com/informalsystems/tendermint-rs/releases).
+
+Releases for each crate are published to crates.io:
+
+- [tendermint][tendermint-docs-link] - Tendermint data structures and
+  serialization
+- [tendermint-rpc][tendermint-rpc-docs-link] - Tendermint RPC client and
+  response types
+
+
+The following crates have not been published to crates.io yet:
+
+- light-client - Tendermint light client library for verifying
+  signed headers, tracking validator set changes, and detecting forks
+- light-node- Tendermint light client binary
+
+## Installation
+
+Requires Rust 1.39+
+
+## Documentation
+
+See documentation on [crates.io](#releases).
```

Comment:
Is this supposed to link to docs.rs? Because currently it just links to above Release section.

Reply:
Release section links to the docs for each crate, figured we didnt need to duplicate for now but in the future we should have more docs in the repo